Masarin.IoT.Sensor/Decoders/MQTTDecoderLoRaWAN.cs

```diff
@@ -64,6 +63,25 @@ public override void Decode(string timestamp, string device, string topic, byte[
                     return;
                 }
             }
+            else if (deviceName.Contains("sn-tcr-01"))
+            {
+                if (obj.ContainsKey("L0_CNT")) {
+                    int[] arr = new int[8] {obj.L0_CNT, obj.L1_CNT, obj.L2_CNT, obj.L3_CNT, obj.R0_CNT, obj.R1_CNT, obj.R2_CNT, obj.R3_CNT};
```

Comment:
Även om jag sa "så enkelt som möjligt" så tycker jag att vi skall hålla isär trafik åt olika håll. Summera L och R var för sig och generera en POST till brokern för vardera riktningen.